src/style.css

```diff
@@ -1,3 +1,146 @@
-body {
-  background-color: lightgray;
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,700;0,900;1,100;1,300&family=Saira:ital,wght@0,100;0,200;0,400;0,500;0,600;1,100;1,300&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "poppins", sans-serif;
+}
+
+header {
+  background-color: rgb(55, 54, 54);
+  color: rgb(178, 178, 178);
+  padding: 1rem;
+}
```

Comment:
Great job styling your project. 👍 However, there are lots of repeated CSS rules and it does not adhere to [HTML & CSS Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 

There is a repeated background color in `header` and `footer` 
- [ ] Kindly consider using this refactoring approach below to improve the maintainability of the code and make it DRY.

   header, footer {
   background-color: rgb(55, 54, 54);
   padding: 1rem;
   color: rgb(178, 178, 178);
   }

   footer {
    width: 100%;
    bottom: 0;
    font-size: 14px;
    }

 This eliminates the need to repeat the background color and color declarations.